hydro_cli/examples/simple_dedalus.hydro.py

```diff
@@ -0,0 +1,65 @@
+import hydro
+import json
+from pathlib import Path
+
+async def main(args):
+    machine_1_gcp = args[0] == "gcp"
+    machine_2_gcp = args[1] == "gcp"
+
+    deployment = hydro.Deployment()
+    localhost_machine = deployment.Localhost()
+
+    machine1 = deployment.GCPComputeEngineHost(
+        project="autocompartmentalization",
+        machine_type="e2-micro",
+        image="debian-cloud/debian-11",
+        region="us-west1-a"
+    ) if machine_1_gcp else localhost_machine
+
+    machine2 = deployment.GCPComputeEngineHost(
+        project="autocompartmentalization",
+        machine_type="e2-micro",
+        image="debian-cloud/debian-11",
+        region="us-west1-a"
+    ) if machine_2_gcp else localhost_machine
+
+    sender = deployment.HydroflowCrate(
+        src=str((Path(__file__).parent / "../../hydro_cli_examples").absolute()),
```

Comment:
If this needs to reference `hydro_cli_examples` why not put it there? Would python not work there?